www/js/TimelineContext.ts

```diff
@@ -50,6 +50,8 @@ type ContextProps = {
   loadMoreDays: (when: 'past' | 'future', nDays: number) => void;
   loadSpecificWeek: (d: string) => void;
   refreshTimeline: () => void;
+  shouldUpdateTimeline: Boolean;
+  setShouldUpdateTimeline: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
For my understanding/edification: Why is only this wrapped in `React.Dispatch` when (say) `setDateRange` is not?